docs/en/settings/spacing-settings.md

```diff
@@ -11,55 +11,55 @@ Vanilla uses numerous spacing variables across the codebase in order to ensure c
 
 Vanilla uses a default spacing unit of `.5rem` (`8px`) as a basis to calculate spacing inside and between components, as well as the line-heights of the different type sizes.
 
-![Baseline grid example](https://assets.ubuntu.com/v1/d2c31b2d-screenshot.png "Baseline grid example")
+![Baseline grid example](https://assets.ubuntu.com/v1/d2c31b2d-screenshot.png 'Baseline grid example')
 
 The image shows the headings sitting on the baseline grid, where the space between each line is one `$sp-unit`.
 
 ### Vertical spacing
 
-The `$spv-intra` variables are used to determine vertical spacing inside components, while `$spv-inter` variables are used for spacing between components.
+The `$spv-inner` variables are used to determine vertical spacing inside components, while `$spv-outer` variables are used for spacing between components.
 
-Spacing variable | Formula | Default value
- ------------------- | ------------- |
-`$spv-intra` | `$sp-unit` | `.5rem`
-`$spv-intra--condensed` | `$spv-intra * .5` | `.25rem`
-`$spv-intra--expanded` | `$spv-intra * 1.5` | `.75rem`
-`$spv-inter--regular` | `$sp-unit * 2` | `1rem`
-`$spv-inter--condensed` | `$sp-unit` | `.5rem`
-`$spv-inter--expanded` | `$sp-unit * 3` | `1.5rem`
+| Spacing variable        | Formula            | Default value |
+| ----------------------- | ------------------ | ------------- |
+| `$spv-inner`            | `$sp-unit`         | `.5rem`       |
+| `$spv-inner--condensed` | `$spv-inner * .5`  | `.25rem`      |
+| `$spv-inner--expanded`  | `$spv-inner * 1.5` | `.75rem`      |
+| `$spv-outer--regular`   | `$sp-unit * 2`     | `1rem`        |
+| `$spv-outer--condensed` | `$sp-unit`         | `.5rem`       |
+| `$spv-outer--expanded`  | `$sp-unit * 3`     | `1.5rem`      |
 
 The following vertical spacing variables are used between a group of components and its wrapper, for example in strips.
 
-Spacing variable | Formula | Default value
- ------------------- | ------------- |
-`$spv-inter--shallow-scaleable` | `$sp-unit * 4` | `2rem`
-`$spv-inter--regular-scaleable` | `$sp-unit * 6` | `3rem`
-`$spv-inter--deep-scaleable` | `$sp-unit * 8` | `4rem`
+| Spacing variable                | Formula        | Default value |
+| ------------------------------- | -------------- | ------------- |
+| `$spv-outer--shallow-scaleable` | `$sp-unit * 4` | `2rem`        |
+| `$spv-outer--regular-scaleable` | `$sp-unit * 6` | `3rem`        |
```

Comment:
@deadlight The scaling variables here are not correct. THey include a static component and a scaleable component. E.g.: 

$spv-inter--regular-scaleable: $sp-unit * 2 * (2 + $multi) !default;

Reply:
@lyubomir-popov looks like this has been wrong for a while. Can we get the renaming landed and then we'll fix this documentation.

Reply:
Sure. Just flagging.

Reply:
Created an issue: #2159